Modules/gcmodule.c

```diff
@@ -2312,3 +2312,23 @@ PyObject_GC_Del(void *op)
     }
     PyObject_FREE(g);
 }
+
+int
+PyObject_GC_IsTracked(void* op_raw)
```

Comment:
Why not directly accepted PyObject* type for the brand new function?

I wrote _PyObject_CAST() for macros and legacy API functions which previously accepted any type.